ironfish-cli/src/commands/wallet/mint.ts

```diff
@@ -230,7 +243,8 @@ export class Mint extends IronfishCommand {
       outputs: [],
       mints: [
         {
-          assetId,
+          // Only provide the asset id if we are not minting an asset for the first time
+          ...(assetData != null ? { assetId } : {}),
```

Comment:
This isn't .. ideal, but the create transaction RPC throws if you pass an asset id for a new asset and I didn't want to muddy this PR up too much